index.html

```diff
@@ -485,47 +490,21 @@ <h4>Questions</h4>
               <div class="container">
                 <div class="section-title center">
                   <h2>Sponsored by</h2>
-                  <hr>
-                  <a href="https://www.facebook.com/toniandguy.vellore/" target=_blank><img src="img/tony_logo.png" height="400" id="tony"></a>
+                  <hr><br>
+                  <a href="#" target=_blank><img src="img/img01.jpg" id="tony"></a>
                   <div class="rules-text" >
-                  </div>
+                  </div><br><br>
```

Comment:
Would prefer using margin/padding instead of repeated `<br>`, Might break the Mobile UI